recipes/CNTK-GPU-OpenMPI/docker/run_convnet_mnist_gpu.sh

```diff
@@ -17,7 +17,7 @@ IFS=',' read -ra HOSTS <<< "$AZ_BATCH_HOST_LIST"
 nodes=${#HOSTS[@]}
 
 # special path for non-mpi job with single gpu
-if [ $nodes -eq 0 ] && [ $ngpus -eq 1 ]; then
```

Comment:
This was intentionally `0`. The path for single node, single GPU is not run with multi-instance tasks, see the [config files](https://github.com/Azure/batch-shipyard/tree/master/recipes/CNTK-GPU-OpenMPI/config/singlenode-singlegpu).